libraries/WiFiClientSecure/src/WiFiClientSecure.h

```diff
@@ -82,6 +82,7 @@ class WiFiClientSecure : public WiFiClient
     bool getFingerprintSHA256(uint8_t sha256_result[32]) { return get_peer_fingerprint(sslclient, sha256_result); };
     int setTimeout(uint32_t seconds);
     int fd() const;
+    int setSocketOption(int option, char* value, size_t len);
```

Comment:
this should not be here, if it's not implemented in the ClientSecure class (it's coming from WiFiClient)